lecture_notes/08 collections/4-concat.js

```diff
@@ -1,22 +1,10 @@
 'use strict';
 
-// Implementation
-
-const getFirstAndLast = (array) => ({
-  first: array[0],
-  last: array[array.length - 1],
-});
-
-const concat = (arr1, arr2) => {
-  const arr = arr1.slice();
-  arr.push(...arr2);
-  return arr;
-};
-
-// Usage
-
 const schoolAges = [10, 12, 15, 15];
 const studentAges = [17, 18, 18, 19, 20];
-const ages = concat(schoolAges, studentAges);
-const { first, last } = getFirstAndLast(ages);
+const NewSet = schoolAges.concat(studentAges);
```

Comment:
Array and Set are different collections, so change confusing names plz
```suggestion
const newSet = schoolAges.concat(studentAges);
```